recipes/tinyply/all/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.4)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

Reply:
3.4 due to https://github.com/ddiakopoulos/tinyply/blob/79a5c65965d193425dd33406a23c028a7aa29544/CMakeLists.txt#L19